src/app/[lang]/about/page.tsx

```diff
@@ -23,7 +23,7 @@ const Page = async () => {
                         />
                     );
                 })}
-            </div>
+            </div>  
```

Comment:
You did not work on this page, so there should not be any changes here.

Add to your vscode settings.json this line:
	"prettier.requireConfig": true,

This will ensure that Prettier formats the code consistently across all team members.